tests/test.sh

```diff
@@ -151,14 +163,18 @@ ret[((index++))]=$?
 ! grep 'DEFINER' mysqldump-php_test012_no-definer.sql
 ret[((index++))]=$?
 
+# test INSERT IGNORE
+diff mysqldump_test013.filtered.sql mysqldump-php_test013.filtered.sql
+ret[((index++))]=$?
+
 rm *.checksum 2> /dev/null
 rm *.filtered.sql 2> /dev/null
 rm mysqldump* 2> /dev/null
 
 echo "Done $index tests"
 
 retvalue=0
-for i in $(seq 0 35) ; do
+for i in $(seq 0 $index) ; do
```

Comment:
I also made a small change here so that the total return code count doesn't have to be adjusted every time a new test is added.